### DISCUSSION TITLE
SERVER_NAME into .env.local

### DISCUSSION THREAD

**Discussion by gponty** [dunglas/symfony-docker#525] [Q&A] [answered]
Hi,

Just for my information why SERVER_NAME variable work into .env file and not in .env.local file ?

To reproduce :
Clone project
Update .env.local with SERVER_NAME
add SERVER_NAME to hosts file
Up docker

(same for XDEBUG_MODE=debug, work only with .env file)
Thank you

**Comment by maxhelias** [comment, marked as answer]
The .env.local file is specific to Symfony. The .env is used not only by Symfony but also by Docker, which uses the variable to send it to the Caddy server.
